packages/dock-blockchain-modules/src/accumulator/internal.js

```diff
@@ -87,6 +87,14 @@ export default class DockInternalAccumulatorModule extends injectParams(
     return (await this.counters(did)).keyCounter;
   }
 
+  async lastParamsId(did) {
+    return await this.paramsCounter(did);
```

Comment:
Without `return await` in case of an Error, this function won't be included in the stack trace making debugging harder.
There was an `eslint` rule called `no-return-await` which is now deprecated, see https://eslint.org/docs/latest/rules/no-return-await and https://github.com/eslint/eslint/issues/17345.

Reply:
fair enough